routes/api/v1/foods.js

```diff
@@ -27,4 +27,30 @@ router.get('/:id', (req, res, next) => {
       }
     })
 })
+
+router.post('/', (req, res, next) => {
+  let food = req.body.food
```

Comment:
If we want to think about refactoring we could possibly pull out some of these error catchers into separate helper functions. Otherwise, all looks good!